pkg/vz/vm_darwin.go

```diff
@@ -334,6 +335,55 @@ func attachDisks(driver *driver.BaseDriver, vmConfig *vz.VirtualMachineConfigura
 	}
 	configurations = append(configurations, diffDisk)
 
+	for _, diskName := range driver.Yaml.AdditionalDisks {
+		d, err := driver.InspectDisk(diskName)
+		if err != nil {
+			return fmt.Errorf("failed to run load disk %q: %q", diskName, err)
+		}
+
+		if d.Instance != "" {
+			return fmt.Errorf("failed to run attach disk %q, in use by instance %q", diskName, d.Instance)
+		}
+		logrus.Infof("Mounting disk %q on %q", diskName, d.MountPoint)
+		err = d.Lock(driver.Instance.Dir)
+		if err != nil {
+			return fmt.Errorf("failed to run lock disk %q: %q", diskName, err)
+		}
+		extraDiskPath := filepath.Join(d.Dir, filenames.DataDisk)
+
+		extraDiskFormat, err := imgutil.DetectFormat(extraDiskPath)
+		if err != nil {
+			return fmt.Errorf("failed to run detect disk format %q: %q", diskName, err)
+		}
+		if extraDiskFormat != "raw" {
+			rawPath := fmt.Sprintf("%s.raw", extraDiskPath)
+			if err = imgutil.QCOWToRaw(extraDiskPath, rawPath); err != nil {
+				return fmt.Errorf("failed to convert qcow2 disk %q to raw for vz driver: %w", diskName, err)
+			}
+			cmd := exec.Command("mv", extraDiskPath, fmt.Sprintf("%s.qcow2", extraDiskPath))
```

Comment:
We could use os Rename() instead of exec mv commands

Reply:
Good call, updated in latest revision